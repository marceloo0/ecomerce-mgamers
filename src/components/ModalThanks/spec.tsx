import { screen } from '@testing-library/react'
import { renderWithTheme } from '../../utils/tests/helpers'

import { ModalThanks } from '.'

jest.mock('../../hooks/useAuth', () => {
  return {
    useAuth() {
      return {
        signout: jest.fn()
      }
    }
  }
})

jest.mock('../../hooks/useCart', () => {
  return {
    useCart() {
      return {
        modalIsOpen: true,
        closeModal: jest.fn()
      }
    }
  }
})

describe('<ModalThanks />', () => {
  it('should render to thanks', () => {
    const { container } = renderWithTheme(<ModalThanks />)

    expect(screen.getByText(/thanks/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
