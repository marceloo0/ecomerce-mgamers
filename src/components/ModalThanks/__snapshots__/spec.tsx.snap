// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalThanks /> should render to thanks 1`] = `
.c0 {
  z-index: 100;
}

<div
  class="c0"
/>
`;
